ark/analysis/spatial_analysis.py

```diff
@@ -294,3 +294,54 @@ def create_neighborhood_matrix(all_data, dist_matrices_dict, included_fovs=None,
         cell_neighbor_freqs.loc[current_fov_neighborhood_data.index, fov_cluster_names] = freqs
 
     return cell_neighbor_counts, cell_neighbor_freqs
+
+
+def compute_neighbor_mat_cluster_scores(neighbor_mat, max_k=10, included_fovs=None,
```

Comment:
How about compute_cluster_metrics

Reply:
Yeah I like that. This function is probably going to be used for more than just neighbor clustering in the future. Will probably think about changing the variable names in the future as well.